### PR TITLE
【テスト】最近の検索：Domain

### LINKED ISSUE #211: 【ベース】Domainのテストの追加

### PR DESCRIPTION
**Pull request by wasabi-muffin**
# 概要
・テストを実装した

# 変更種類
- [ ] フィーチャー実装
- [ ] バッグ修正
- [ ] リファクタ
- [ ] ドキュメンテーション
- [ ] その他（）

# テスト
- [ ] Unit Test
- [ ] UI Test
- [ ] 動作確認
- [ ] N/A

### CODE REVIEW

File: domain/src/test/kotlin/jp/co/yumemi/android/code_check/domain/usecases/ClearRecentSearchesExecutorTest.kt [hunk 1]
```diff
@@ -0,0 +1,38 @@
+package jp.co.yumemi.android.code_check.domain.usecases
+
+import io.kotest.matchers.shouldBe
```
Comment (gmvalentino8) [inline domain/src/test/kotlin/jp/co/yumemi/android/code_check/domain/usecases/ClearRecentSearchesExecutorTest.kt:3]: <table data-meta="generated_by_./Dangerfile">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/gmvalentino8/github-sample-project/blob/3df85becc55eed72b2ca0dd93139cdd5cba0ea34/domain/src/test/kotlin/jp/co/yumemi/android/code_check/domain/usecases/ClearRecentSearchesExecutorTest.kt#L3"></span>Unused import</td>
    </tr>
  </tbody>
</table>

File: domain/src/test/kotlin/jp/co/yumemi/android/code_check/domain/usecases/ClearRecentSearchesExecutorTest.kt [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+import io.kotest.matchers.types.shouldBeTypeOf
```
Comment (gmvalentino8) [inline domain/src/test/kotlin/jp/co/yumemi/android/code_check/domain/usecases/ClearRecentSearchesExecutorTest.kt:4]: <table data-meta="generated_by_./Dangerfile">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/gmvalentino8/github-sample-project/blob/3df85becc55eed72b2ca0dd93139cdd5cba0ea34/domain/src/test/kotlin/jp/co/yumemi/android/code_check/domain/usecases/ClearRecentSearchesExecutorTest.kt#L4"></span>Unused import</td>
    </tr>
  </tbody>
</table>

File: domain/src/test/kotlin/jp/co/yumemi/android/code_check/domain/usecases/ClearRecentSearchesExecutorTest.kt [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+import io.mockk.coEvery
+import io.mockk.coVerify
+import io.mockk.mockk
+import jp.co.yumemi.android.code_check.domain.core.DomainResult
```
Comment (gmvalentino8) [inline domain/src/test/kotlin/jp/co/yumemi/android/code_check/domain/usecases/ClearRecentSearchesExecutorTest.kt:8]: <table data-meta="generated_by_./Dangerfile">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/gmvalentino8/github-sample-project/blob/3df85becc55eed72b2ca0dd93139cdd5cba0ea34/domain/src/test/kotlin/jp/co/yumemi/android/code_check/domain/usecases/ClearRecentSearchesExecutorTest.kt#L8"></span>Unused import</td>
    </tr>
  </tbody>
</table>

File: domain/src/test/kotlin/jp/co/yumemi/android/code_check/domain/usecases/ClearRecentSearchesExecutorTest.kt [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+import jp.co.yumemi.android.code_check.domain.core.ErrorHandler
```
Comment (gmvalentino8) [inline domain/src/test/kotlin/jp/co/yumemi/android/code_check/domain/usecases/ClearRecentSearchesExecutorTest.kt:9]: <table data-meta="generated_by_./Dangerfile">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/gmvalentino8/github-sample-project/blob/3df85becc55eed72b2ca0dd93139cdd5cba0ea34/domain/src/test/kotlin/jp/co/yumemi/android/code_check/domain/usecases/ClearRecentSearchesExecutorTest.kt#L9"></span>Unused import</td>
    </tr>
  </tbody>
</table>

File: domain/src/test/kotlin/jp/co/yumemi/android/code_check/domain/usecases/ClearRecentSearchesExecutorTest.kt [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+import jp.co.yumemi.android.code_check.domain.core.Pageable
```
Comment (gmvalentino8) [inline domain/src/test/kotlin/jp/co/yumemi/android/code_check/domain/usecases/ClearRecentSearchesExecutorTest.kt:10]: <table data-meta="generated_by_./Dangerfile">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/gmvalentino8/github-sample-project/blob/3df85becc55eed72b2ca0dd93139cdd5cba0ea34/domain/src/test/kotlin/jp/co/yumemi/android/code_check/domain/usecases/ClearRecentSearchesExecutorTest.kt#L10"></span>Unused import</td>
    </tr>
  </tbody>
</table>

File: domain/src/test/kotlin/jp/co/yumemi/android/code_check/domain/usecases/ClearRecentSearchesExecutorTest.kt [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+import jp.co.yumemi.android.code_check.domain.entities.Issue
```
Comment (gmvalentino8) [inline domain/src/test/kotlin/jp/co/yumemi/android/code_check/domain/usecases/ClearRecentSearchesExecutorTest.kt:11]: <table data-meta="generated_by_./Dangerfile">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/gmvalentino8/github-sample-project/blob/3df85becc55eed72b2ca0dd93139cdd5cba0ea34/domain/src/test/kotlin/jp/co/yumemi/android/code_check/domain/usecases/ClearRecentSearchesExecutorTest.kt#L11"></span>Unused import</td>
    </tr>
  </tbody>
</table>

File: domain/src/test/kotlin/jp/co/yumemi/android/code_check/domain/usecases/ClearRecentSearchesExecutorTest.kt [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+import jp.co.yumemi.android.code_check.domain.entities.RecentSearch
```
Comment (gmvalentino8) [inline domain/src/test/kotlin/jp/co/yumemi/android/code_check/domain/usecases/ClearRecentSearchesExecutorTest.kt:12]: <table data-meta="generated_by_./Dangerfile">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/gmvalentino8/github-sample-project/blob/3df85becc55eed72b2ca0dd93139cdd5cba0ea34/domain/src/test/kotlin/jp/co/yumemi/android/code_check/domain/usecases/ClearRecentSearchesExecutorTest.kt#L12"></span>Unused import</td>
    </tr>
  </tbody>
</table>

**Comment by wasabi-muffin** [comment]
No error was reported but at least one warning was found.

<p align="right" data-meta="generated_by_./Dangerfile">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Pull request merged**